packages/sui-mono/bin/sui-mono-changelog.js

```diff
@@ -30,21 +28,20 @@ program
   .parse(process.argv)
 
 const CHANGELOG_NAME = getChangelogFilename()
-const cwd = path.join(process.cwd(), getPackagesFolder())
-const getRepoFolders = () =>
-  !isMonoPackage() ? getPackagesPaths(cwd)(getScopes()) : ['.']
-const folders = program.args.length ? program.args : getRepoFolders()
+
+const folders = program.args.length ? program.args : getWorkspaces()
```

Comment:
do you know where we are using this program.args use case?

Reply:
No idea, tbh.